src/pages/NotFound.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default function NotFound() {
+  return <main className="flex-1">Not Found</main>;
+}
```

Comment:
NotFound 페이지에 Header와 Footer를 넣어주면 사용자 편의성이 증가할 것입니다.
혹은 PR에 작성하신 것처럼 메인 페이지로 돌아갈 수 있는 장치가 있으면 좋을 것 같습니다.

```suggestion
export default function NotFound() {
  return (
    <>
      <Header />
      <main className="flex-1">Not Found</main>
      <Footer />
    <>
  );
}
```

Reply:
그쵸 이 부분 처음에 당황 했었습니다. 에러페이지도 당연히 <Outlet />에 의해 헤더, 푸터가 고정될 줄 알았는데 안되더라고요.. 그래서 아래의 글을 보고 해결했습니다~

https://github.com/remix-run/react-router/discussions/9553#discussioncomment-4907035